### PR TITLE
port sentiment analysis to new UI framework

### PR DESCRIPTION
**Pull request by epwalsh**
Still TODO:
- [x] Add interpret section
- [x] Add attackers
- [ ] ~~Add model selection (only does glove right now)~~ never mind
- [ ] Fix this formatting in model card info. Seeing raw markdown. Not sure if this is something that should be fixed on the UI or the backend.
image

This should be reviewed very carefully when it's ready because I have no idea what I'm doing with front-end code.

### CODE REVIEW

**Comment by epwalsh** [comment]
After this merges I am officially a full-stack developer. cc: @schmmd

**Comment by jonborchardt** [comment]
@epwalsh the formatting issue will be taken care of by @AkshitaB 
this code looks good enough to ship... assuming it works!

**Review by jonborchardt — APPROVED** [review]

**Comment by jonborchardt** [comment]
why are we not doing glove?
need attacks for glove

File: ui/src/demos/sentiment-analysis/Main.tsx [hunk 1]
```diff
@@ -1,17 +1,79 @@
...
+                                                inputHeaders={['Sentence']}
+                                            />
+                                        )}
+                                    </Interpreters>
+                                    // TODO: [jon] this needs to be "grad_input_1" here
+                                    <Attackers
+                                        input={input}
+                                        model={model}
+                                        prediction={output}
+                                        target="tokens"
```
Comment (epwalsh) [inline ui/src/demos/sentiment-analysis/Main.tsx:66]: I don't know what the issue is with this line

Comment (epwalsh) [reply ui/src/demos/sentiment-analysis/Main.tsx:66]: Oh, weird, it only failed to compile locally.

**Review by jonborchardt — APPROVED** [review]

**Pull request merged**